### ISSUE TITLE
What's the point of return const scalar type?

### ISSUE THREAD

**Issue by nnop** [colmap/pycolmap#157]
https://github.com/colmap/pycolmap/blob/b44ffd38573016fcc28636c37ba2680033f1665b/estimators/generalized_absolute_pose.cc#L28-L46

Is there any consideration for returning a const scalar type? In my understanding, it'd be copied anyway.

**Comment by sarlinpe** [comment]
Good point. Irrelevant since https://github.com/colmap/pycolmap/commit/67d84e197129f49c3c9552405ac1cf6901a4de5a.
